app/views/modules/_footer.html.erb

```diff
@@ -43,6 +43,10 @@ Unless required by applicable law or agreed to in writing, software distributed
 	</div>
         <div class="row copyright">&copy; <a href="https://library.ualberta.ca">University of Alberta Libraries</a> | University of Alberta, Edmonton, AB, Canada T6G 2R3 | <a href="https://library.ualberta.ca/about-us/policies/privacy-policy">Web Site Privacy Policy</a>
 	</div>
+    <div class="row avalon">
+      <a href="http://www.avalonmediasystem.org">Powered by Avalon</a>
+      <!-- Avalon Media System Release <%= Avalon::VERSION %> -->
+	  </div>
```

Comment:
Yes, the view in avalon is a sorry mess of bad indentation (including mixed tabs/spaces in some spots). Will fix.